rfcs/property-readonly.md

```diff
@@ -0,0 +1,165 @@
+# Read-only properties
+
+## Summary
+
+Allow properties of classes and tables to be marked as read-only.
+
+## Motivation
+
+Currently, Roblox APIs have read-only properties of classes, but our
+type system does not track this. As a result, users can write (and
+indeed due to autocomplete, an encouraged to write) programs with
+run-time errors.
+
+In addition, user code may have properties (such as module exports)
+that are expected to be used without modification. Currently there is
+no way for user code to indicate this, even if it has explicit type
+annotations.
```

Comment:
Module exports were removed below, so maybe this paragraph needs an update.

Reply:
Done.